solvebio/resource/vault.py

```diff
@@ -0,0 +1,70 @@
+"""Solvebio Vault API resource"""
+# from ..client import client
+from ..help import open_help
+
+from .apiresource import CreateableAPIResource
+from .apiresource import ListableAPIResource
+from .apiresource import SearchableAPIResource
+from .apiresource import UpdateableAPIResource
+from .apiresource import DeletableAPIResource
+
+
+class Vault(CreateableAPIResource,
```

Comment:
Good idea.  I've added a `.objects` property which calls out to the `.all` method with the appropriate `vault_id` filter.  

```
+    vault = Vault.get_personal_vault()
+    vault.objects.all(path='/1000G')
+    vault.objects.all(query='GRC')
```

https://github.com/solvebio/solvebio-python/blob/f-hyrule/solvebio/resource/vault.py#L11

https://github.com/solvebio/solvebio-python/blob/f-hyrule/solvebio/resource/vault.py#L79